force_wfmanager/ui/setup/mco/kpi_specification_model_view.py

```diff
@@ -26,8 +26,12 @@ class KPISpecificationModelView(BaseMCOOptionsModelView):
     # model.name listed in kpi_names. However, it is possible
     # to directly change model.name without updating kpi_names
     traits_view = View(
-        Item('name', object='model',
-             editor=EnumEditor(name='object._combobox_values')),
+        Item('selected_variable',
```

Comment:
Rather than using an `EnumEditor`, we can select between `Variable` instances from the list `available_variables`